sql/group_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package sql
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/util/leaktest"
+)
+
+func TestDesiredAggregateOrder(t *testing.T) {
+	defer leaktest.AfterTest(t)
+
+	testData := []struct {
+		expr     string
+		ordering []int
+	}{
+		{`a`, nil},
+		{`MIN(a)`, []int{1}},
+		{`MAX(a)`, []int{-1}},
+		{`(MIN(a), MAX(a))`, nil},
+		{`(MIN(a), AVG(a))`, nil},
+		{`(MIN(a), COUNT(a))`, nil},
+		{`(MIN(a), SUM(a))`, nil},
+		{`(MIN(a+1), MIN(a))`, nil},
```

Comment:
wait, so does that mean that `MIN(a), MIN(a)` should not return `nil`? I think it currently does.